src/dedupe-hub/api/app/utils/rollbar.py

```diff
@@ -9,13 +9,16 @@
 def init_rollbar():
     if not DEBUG:
         rollbar.init(**ROLLBAR)
-
+            
```

Comment:
this seems a formatting issue

Reply:
My editor always adds a new line at the end of the files. Shall I remove this rule for Python?

Reply:
As I remember end file line is correct behavior.

Reply:
Will check, thanks